server/schemas/schemaModel.js

```diff
@@ -14,40 +14,46 @@ bluebird.promisifyAll(Model);
 module.exports = {
   model: Model,
 
+  /**
+   *createTemplate:
+   * @param {Object} request the http ClientRequest object
+   * @param {Object} response the http ServerResponse object
+   * @param {Function} next callback to be executed with error and schema arguments.
+   * Accepts incoming request, response, and next (callback).
+   * Attempts to find an existing schema with the provided name. If one does not exists,
+   * a schema template is created from the request parameters.
+  */
```

Comment:
i don't think this is valid. did you run jsdoc on the project and look at the result?